xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/documentTags.vm

```diff
@@ -131,9 +131,13 @@
         #else
             ## TODO
         #end
+    #else
+       #if($tags.size() == 0)
```

Comment:
why not `$tags.isEmpty()` which is the traditional way of doing it?

Reply:
In new commit I use this